faqs.rst

```diff
@@ -256,20 +256,30 @@ systems, ``/tmp`` is a RAM-backed file system with limited space, and in
 some situations such as long running, high throughput workflows, Flux may
 use a lot of it.
 
+When the FLux database fills up the disk, errors like the following may
+appear and the instance of Flux will get stuck or otherwise not function
+properly
+
+.. code-block:: console
+
+ content-sqlite.err[0]: store: executing stmt: database or disk is full(13)
```

Comment:
Nit: can we be consistent with spacing for code blocks? This one has indentation that is much smaller than the others.

Reply:
Sorry I copied the example right before this code block which has an indent of 1. There are also code blocks in this file with indent of 2, 3, and 4. I'll tack on an extra commit to standardize on 4.